resources/metadata/transportRequestUploadSOLMAN.yaml

```diff
@@ -0,0 +1,92 @@
+metadata:
+  name: transportRequestUploadSOLMAN
+  aliases:
+    - name: transportRequestUploadFile
+  description: "Uploads content to a transport request"
```

Comment:
I kept the short description since it is used in `piper -help` and for that reason it should be short. The "solman" can also be derived from the step name. Instead there is now some more text in the long description, which also references SOLMAN now.